lld/test/COFF/common-dedup.ll

```diff
@@ -0,0 +1,38 @@
+; REQUIRES: x86
+; RUN: split-file %s %t.dir
+; RUN: llc %t.dir/t1.ll -o %t.t1.obj --filetype=obj
```

Comment:
@DavidSpickett is strongly of the opinion that mixing `%t.dir/...` and `%t.<suffix>` is a bad inconsistency, and if using `%t.dir` we should be using it for everything. Personally, I don't mind the mix.